controllers/core/cninode_controller.go

```diff
@@ -0,0 +1,101 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
Why is this inside /core/? Looks like they're partially ordered by API group, but CNINode isn't part of code.

Reply:
Make sense. API should be a better place for CRDs.